earn/src/pages/BoostPage.tsx

```diff
@@ -5,43 +5,75 @@ import { UniswapV3PoolABI } from 'shared/lib/abis/UniswapV3Pool';
 import AppPage from 'shared/lib/components/common/AppPage';
 import { Text } from 'shared/lib/components/common/Typography';
 import { Token } from 'shared/lib/data/Token';
-import { toBig } from 'shared/lib/util/Numbers';
-import { useAccount, useContractReads, useProvider } from 'wagmi';
+import { Address, useAccount, useContractReads, useProvider } from 'wagmi';
 
 import { ChainContext } from '../App';
 import BoostCard from '../components/boost/BoostCard';
 import { BoostCardPlaceholder } from '../components/boost/BoostCardPlaceholder';
 import ManageBoostModal from '../components/boost/ManageBoostModal';
-import { sqrtRatioToPrice } from '../data/BalanceSheet';
 import { fetchBoostBorrower, fetchBoostBorrowersList } from '../data/Uniboost';
 import {
   UniswapNFTPosition,
   UniswapPosition,
   computePoolAddress,
   fetchUniswapNFTPositions,
   getAmountsForLiquidity,
-  tickToPrice,
+  getValueOfLiquidity,
 } from '../data/Uniswap';
 import { getProminentColor } from '../util/Colors';
 
-export type UniswapNFTCardInfo = {
-  token0: Token;
-  token1: Token;
-  minPrice: number;
-  maxPrice: number;
-  amount0: number;
-  amount1: number;
-  amount0Percent: number;
-  amount1Percent: number;
-  isInRange: boolean;
-  isDeposit: boolean;
-  poolAddress: string;
-  color0: string;
-  color1: string;
-  minTick: number;
-  maxTick: number;
-  currentTick: number;
-};
+export enum BoostCardType {
+  UNISWAP_NFT,
+  BOOST_NFT,
+  BOOST_NFT_GENERALIZED,
+}
```

Comment:
Switched to enum instead of the `isDeposit` boolean since there are currently 3 states and likely more in the future (ex: BOOST_NFT_LIQUIDATED)